pkg/cloud/vsphere/actuators/cluster/actuator.go

```diff
@@ -115,47 +114,6 @@ func (a *Actuator) Delete(cluster *clusterv1.Cluster) (opErr error) {
 	return nil
 }
 
-// GetIP returns the control plane endpoint for the cluster.
-func (a *Actuator) GetIP(cluster *clusterv1.Cluster, machine *clusterv1.Machine) (string, error) {
-	clusterContext, err := context.NewClusterContext(&context.ClusterContextParams{
-		Cluster:    cluster,
-		Client:     a.client,
-		CoreClient: a.coreClient,
-	})
-	if err != nil {
-		return "", err
-	}
-	machineContext, err := context.NewMachineContextFromClusterContext(clusterContext, machine)
-	if err != nil {
-		return "", err
-	}
-	return machineContext.ControlPlaneEndpoint()
-}
-
-// GetKubeConfig returns the contents of a Kubernetes configuration file that
-// may be used to access the cluster.
-func (a *Actuator) GetKubeConfig(cluster *clusterv1.Cluster, machine *clusterv1.Machine) (string, error) {
-	clusterContext, err := context.NewClusterContext(&context.ClusterContextParams{
-		Cluster:    cluster,
-		Client:     a.client,
-		CoreClient: a.coreClient,
-	})
-	if err != nil {
-		return "", err
-	}
-
-	if machine == nil {
```

Comment:
Do we need an equivalent machine == nil check here in the Deployer?

Reply:
The deployer always existed, so it shouldn’t need to be changed. But I’m happy to add it.

Reply:
Fixed.

Reply:
Thought cluster actuator calls this but and needed the machine == nil check, looks like that changed recently though